imageproxy.go

```diff
@@ -65,7 +65,7 @@ type Proxy struct {
 	Logger *log.Logger
 
 	// SignatureKey is the HMAC key used to verify signed requests.
-	SignatureKey []byte
+	SignatureKey [][]byte
```

Comment:
probably rename field to `SignatureKeys` to reflect the change?  Also update godoc to reflect that this contains multiple keys, any of which can be used for signatures.